src/Modal/Modal.js

```diff
@@ -0,0 +1,73 @@
+/** @jsx jsx */
+import ReactModal from 'react-modal';
+import PropTypes from 'prop-types';
+import omit from 'lodash/omit';
+import { Fragment } from 'react';
+import { Global, css, jsx } from '@emotion/core';
+import { transparentize } from 'polished';
+import { withTheme } from 'emotion-theming';
+
+const baseOverlayCss = theme => css`
+  align-items: center;
+  background: ${transparentize(0.72, theme.colors.grey100)};
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+`;
+
+const baseModalCss = theme => css`
+  align-items: center;
+  box-sizing: border-box;
+  display: flex;
+  height: calc(100% - ${theme.space.largest});
+  justify-content: center;
+  outline: none;
+  width: calc(100% - ${theme.space.largest});
+`;
+
+const Modal = ({ children, modalCss, overlayCss, ...props }) => {
+  const combinedModalCss = theme => css`
+    .ArborModalOverlay {
+      ${baseOverlayCss(theme)};
+      ${typeof overlayCss === 'function' ? overlayCss(theme) : overlayCss};
+    }
+
+    .ArborModal {
+      ${baseModalCss(theme)};
+      ${typeof modalCss === 'function' ? modalCss(theme) : modalCss};
+    }
+  `;
+
+  return (
+    <Fragment>
+      <Global styles={combinedModalCss} />
+      <ReactModal
+        overlayClassName="ArborModalOverlay"
+        className="ArborModal"
+        {...omit(props, 'theme')}
+      >
+        {children}
+      </ReactModal>
+    </Fragment>
+  );
+};
+
+Modal.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.node,
+    PropTypes.arrayOf(PropTypes.node)
+  ]).isRequired,
+  modalCss: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+  overlayCss: PropTypes.oneOfType([PropTypes.string, PropTypes.func])
```

Comment:
I feel like when we were pairing we ran into something where these CSS props weren’t ever actually strings. Is that right?

Reply:
mmm, good Q. Ill have to go back in and look